### PR TITLE
Polish Calendar UI (month view)

### PR DESCRIPTION
**Pull request by johnjovero98**
## What issue is this referencing?

#394 

## Do these code changes work locally and have you tested that they fix the issue yourself?

-   [x] yes!

## Does the following command run without warnings or errors?

-   [x] yes! `npm run pr-checks`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [x] yes!

## My node version matches the one suggested when running `nvm use`?

-   [x] yes!

Here is a demo gif of the calendar redesign. 

**Note:** I used pure CSS to make the changes instead of tailwind classes. I'm not sure if that will cause any performance issues. Please let me know if I need to refactor it :>

month-view-demo

### CODE REVIEW

File: components/app/Calendar.vue [hunk 1]
```diff
@@ -35,7 +33,7 @@ const onEventClick = (event: any, e: any) => {
       :events-on-month-view="true"
       :twelve-hour="true"
       :events="group.items"
-      :start-week-on-sunday="false"
+      :start-week-on-sunday="true"
```
Comment (dgmouris) [inline components/app/Calendar.vue:36]: We might have to rebase this branch because due to another PR that put this in.

File: nuxt.config.ts [hunk 2]
```diff
@@ -82,6 +81,8 @@ export default defineNuxtConfig({
     componentIslands: true,
   },
 
+  compatibilityDate: '2024-10-18',
```
Comment (dgmouris) [inline nuxt.config.ts:84]: I'm not sure what this is for, was this needed?

**Review by dgmouris — APPROVED** [review]
Looks good but we should rebase and remove the `compabilityDate` if not needed.

File: components/app/Calendar.vue (hunk 1 above)
Comment (dgmouris) [reply components/app/Calendar.vue:36]: technically it won't overwrite because it's the same line.

File: nuxt.config.ts (hunk 2 above)
Comment (arashsheyda) [reply nuxt.config.ts:84]: this is from [Nuxt](https://nuxt.com/docs/api/nuxt-config#compatibilitydate). we don't necessarily need to include it in the PR.

Comment (johnjovero98) [reply nuxt.config.ts]: I removed line 84 and got this branch up to date :>  

Screenshot 2024-10-21 at 3 21 09 PM

**Comment by dgmouris** [comment]
Thanks @johnjovero98 this is great I'm merging this.

**Comment by MandyMeindersma** [comment]
@allcontributors add @johnjovero98 for code

**Pull request merged**